pkg/volume/glusterfs/glusterfs.go

```diff
@@ -822,11 +822,11 @@ func (p *glusterfsVolumeProvisioner) CreateVolume(gid int) (r *v1.GlusterfsVolum
 		return nil, 0, "", fmt.Errorf("failed to get cluster nodes for volume %s: %v", volume, err)
 	}
 
-	// The 'endpointname' is created in form of 'glusterfs-dynamic-<claimname>'.
+	// The 'endpointname' is created in form of 'glusterfs-dynamic-<PVC UUID>'.
 	// createEndpointService() checks for this 'endpoint' existence in PVC's namespace and
-	// If not found, it create an endpoint and service using the IPs we dynamically picked at time
+	// if not found, it create an endpoint and service using the IPs we dynamically picked at time
```

Comment:
This is correct but actually not related to the topic of this patch.
So that comment fix might be a separate patch...

Reply:
Put it as  a seperate commit.

Reply:
Thanks @humblec . I noticed there is another error in the line: `it create` --> `it creates`. Maybe we just skip this hunk from the PR and consider the separate PR #70306 for comment cleanup?